modoboa_installer/scripts/dovecot.py

```diff
@@ -30,6 +30,16 @@ class Dovecot(base.Installer):
         "conf.d/10-master.conf", "conf.d/20-lmtp.conf", "conf.d/10-ssl-keys.try"]
     with_user = True
 
+    def setup_user(self):
+        """Setup mailbox user."""
+        self.mailboxes_owner = self.app_config["mailboxes_owner"]
+        if self.config.has_option(self.appname, "home_dir"):
+            self.home_dir = self.config.get(self.appname, "home_dir")
+        else:
+            self.home_dir = None
+        system.create_user(self.mailbox_owner, self.home_dir)
+        super(Dovecot, self).setup_user()
```

Comment:
```super().setup_user()``` is enough with Python3

Reply:
And actually you could rewrite this method like this:
```python
def setup_user(self):
    """Setup mailbox user."""
    super().setup_user()
    self.mailboxes_owner = self.app_config["mailboxes_owner"]
    system.create_user(self.mailbox_owner, self.home_dir)
```

Reply:
I just wanted to keep the model used throughout other super calls. Do you want to update them all?